README.md

```diff
@@ -1,57 +1,75 @@
-# Project Name
-
-(short, 1-3 sentenced, description of the project)
+---
+page_type: sample
+languages:
+- javascript
+products:
+- nodejs
+- ms-graph
+- azure-active-directory
+description: "Demonstrates how to use MSAL Node to acquire an access token for a protected resource such as Microsoft Graph in a console application using the application's own identity (client credentials flow)."
+urlFragment: "ms-identity-javascript-nodejs-console"
+---
+
+# A Node.js console application secured by MSAL Node (Preview) on Microsoft identity platform
+
+This sample demonstrates how to use [MSAL Node (Preview)](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/lib/msal-node) to acquire an access token for a protected resource such as Microsoft Graph in a console application using the application's own identity ([client credentials flow](https://docs.microsoft.com/azure/active-directory/develop/v2-oauth2-client-creds-grant-flow)).
 
 ## Features
 
-This project framework provides the following features:
+This sample demonstrates the following **MSAL Node** concepts:
 
-* Feature 1
-* Feature 2
-* ...
+* Configuration
+* Acquiring an access token
+* Calling a web API
 
-## Getting Started
+## Contents
 
-### Prerequisites
+| File/folder           | Description                                                  |
+|-----------------------|--------------------------------------------------------------|
+| `AppCreationScripts/` | Contains Powershell scripts for automating app registration. |
+| `bin/index.js`        | Application entry.                                           |
+| `bin/auth.js`         | Main authentication logic resides here.                      |
+| `bin/fetch.js`        | Contains an Axios HTTP client for calling endpoints.         |
+| `.env`                | Environment variables of authentication parameters.          |
 
-(ideally very short, if any)
+## Getting Started
 
-- OS
-- Library version
-- ...
+### Prerequisites
 
-### Installation
+* [Node.js](https://nodejs.org/en/) must be installed to run this sample.
+* [Visual Studio Code](https://code.visualstudio.com/download) is recommended for running and editing this sample.
 
-(ideally very short)
+### Setup
 
-- npm install [package name]
-- mvn install
-- ...
+1. [Register a new application](https://docs.microsoft.com/azure/active-directory/develop/scenario-daemon-app-registration) in the [Azure Portal](https://portal.azure.com). Ensure that you have obtained **admin consent** for API permissions.
```

Comment:
```suggestion
1. Register [a Daemon app that calls web APIs](https://docs.microsoft.com/azure/active-directory/develop/scenario-daemon-app-registration) in the [Azure Portal](https://portal.azure.com).
1. In the app's registration screen, select the API permissions blade in the left to open the page where we add access to the APIs that your application needs.
    - Select the Add a permission button and then,
    - Ensure that the Microsoft APIs tab is selected.
    - In the Commonly used Microsoft APIs section, select Microsoft Graph
    - Select Application Permissions. 
    - In this section, select the `User.Read.All` from the list. Use the search box if necessary.
    - Select the Add permissions button at the bottom.
    - Be sure to click the checkmark to **Grant admin consent**.
```

Reply:
+1, its "Application Permissions"

---
In reply to: [560588228](https://github.com/Azure-Samples/ms-identity-javascript-nodejs-console/pull/1#discussion_r560588228) [](ancestors = 560588228)